engine/src/main/java/com/redhat/cloud/notifications/templates/ResourceOptimizationService.java

```diff
@@ -0,0 +1,40 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.redhat.cloud.notifications.models.EmailSubscriptionType;
+import io.quarkus.qute.CheckedTemplate;
+import io.quarkus.qute.TemplateInstance;
+
+// Name needs to be "ResourceOptimizationService" to read templates from resources/templates/ResourceOptimizationService
+public class ResourceOptimizationService implements EmailTemplate {
+
+    @Override
+    public TemplateInstance getTitle(String eventType, EmailSubscriptionType type) {
+
+        return Templates.dailyEmailTitle();
+    }
+
+    @Override
+    public TemplateInstance getBody(String eventType, EmailSubscriptionType type) {
+
+        return Templates.dailyEmailBody();
+    }
+
+    @Override
+    public boolean isSupported(String eventType, EmailSubscriptionType type) {
+        return type == EmailSubscriptionType.DAILY;
+    }
+
+    @Override
+    public boolean isEmailSubscriptionSupported(EmailSubscriptionType type) {
+        return type == EmailSubscriptionType.DAILY;;
```

Comment:
This will fix the build:
```suggestion
        return type == EmailSubscriptionType.DAILY;
```

Reply:
Oops! Fixing it. Thanks!